lib/plugins/aws/deploy/lib/cleanupEcrRepository.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const _ = require('lodash');
+
+module.exports = {
+  async cleanupEcrRepository() {
+    // TODO: Add check if we should even try removing anything
+    const repositoryName = this.provider.naming.getEcrRepositoryName();
+    const registryId = await this.provider.getAccountId();
```

Comment:
Do you see any issues with memoize'ing `getAccountId` call on provider? I'm using it a few times and I dont' want to incur extra API calls - I think it's safe but wanted to double check here

Reply:
I think it should be totally safe.  I do not envision scenario where creds change over the course of processing command.

Still there might be an issue if this command is invoked prematurely (e.g. before variables are resolved), but hopefully there won't be such case.

Also memoizing may influence tests. If there are issues, it might be needed to reset cache here: https://github.com/serverless/serverless/blob/b4fef7da64978433b88ee9fc8e9544e80608a387/test/mochaPatch.js#L22-L28 but I think, that in this case it should not be needed